tests/test_app/Plugin/Named/src/NamedPlugin.php

```diff
@@ -1,17 +1,17 @@
 <?php
 declare(strict_types=1);
 
-namespace ParentPlugin;
+namespace Named;
 
 use Cake\Core\BasePlugin;
 use Cake\Core\Configure;
 use Cake\Core\PluginApplicationInterface;
 
-class Plugin extends BasePlugin
+class NamedPlugin extends BasePlugin
 {
     public function bootstrap(PluginApplicationInterface $app): void
     {
-        Configure::write('ParentPlugin.bootstrap', true);
+        Configure::write('Named.bootstrap', true);
```

Comment:
I don't see any changes for this name lookup. Is it dependent on anything?

Reply:
See the BaseApplicationTest.